model/src/main/java/userinfo/UserInfo.java

```diff
@@ -0,0 +1,42 @@
+package userinfo;
+
+/**
+ * Stores the information of a user.
+ */
+public abstract class UserInfo {
+    private String address;
+    private String phoneNumber;
```

Comment:
These probably belong in subclasses rather than here, there is no guarantee every type of `UserInfo` will have both these fields; however, we may want some form of email field either here, or in the `User` entity object itself. You should collaborate with whoever is working on #9 to figure out which approach you will take.